lib/sevak/core.rb

```diff
@@ -20,6 +20,10 @@ def connection
         attempt += 1
         sleep(0.001)
         retry if attempt < 10
+      rescue Bunny::ConnectionClosedError => e
+        attempt += 1
+        sleep(0.001)
+        retry if attempt < 10
```

Comment:
Change the line as follows. Because we need to throw the error if the attempt > 10 .
attempt < 10 ? retry : raise e